### DISCUSSION TITLE
Is UDP transport supported by libp2p?

### DISCUSSION THREAD

**Discussion by stepa2** [libp2p/rust-libp2p#2204] [Q&A]
Is UDP transport supported by libp2p?
If not, are there any alternatives in UDP P2P networking?

**Comment by thomaseizinger** [comment]
There is an in-progress QUIC transport that builds on top of UDP.

Native UDP is unsupported by libp2p afaik because every abstraction on top relies on a reliable transport.

**Reply by mxinden** [reply]
Additional context to @thomaseizinger reply above: https://github.com/libp2p/specs/blob/master/ROADMAP.md#-message-oriented-transports
